projects/1.20/assets/modrinth-deepdrilling/deepdrilling/lang/zh_cn.json

```
@@ -0,0 +1,63 @@
+  "block.deepdrilling.collection_filter": "收集模块",
+  "block.deepdrilling.collection_filter.tooltip.behaviour1": "能够_收集_本会_掉落在地上_的产物。",
+  "block.deepdrilling.collection_filter.tooltip.behaviour2": "可取出_采掘出的产物_",
+  "block.deepdrilling.collection_filter.tooltip.condition1": "连接至钻芯时",
+  "block.deepdrilling.collection_filter.tooltip.condition2": "右键单击此模块",
+  "block.deepdrilling.collection_filter.tooltip.summary": "一种_钻探模块_，可与_钻芯_连接。",
+  "block.deepdrilling.copper_drill_head": "铜制钻头",
+  "block.deepdrilling.crimsite_node": "绯红岩矿脉",
+  "block.deepdrilling.crimsite_node.tooltip.behaviour1": "_绯红岩，凝灰岩，石灰岩_",
+  "block.deepdrilling.crimsite_node.tooltip.condition1": "表面沉积物",
+  "block.deepdrilling.crimsite_node.tooltip.summary": "在_山地生物群系_的_基岩层_中生成",
+  "block.deepdrilling.drill_core": "钻芯",
+  "block.deepdrilling.drill_overclock": "超频模块",
+  "block.deepdrilling.drill_overclock.tooltip.behaviour1": "_提高采掘效率_的同时_大幅增加钻头的损耗_。",
+  "block.deepdrilling.drill_overclock.tooltip.condition1": "连接至钻芯时",
+  "block.deepdrilling.drill_overclock.tooltip.summary": "一种_钻探模块_，可与_钻芯_连接。",
+  "block.deepdrilling.ochrum_node": "赭金砂矿脉",
+  "block.deepdrilling.ochrum_node.tooltip.behaviour1": "_赭金砂，凝灰岩，滴水石块_",
+  "block.deepdrilling.ochrum_node.tooltip.condition1": "表面沉积物",
+  "block.deepdrilling.ochrum_node.tooltip.summary": "在_沙漠生物群系_的_基岩层_中生成",
+  "block.deepdrilling.sludge": "淤泥",
+  "block.deepdrilling.sludge_pump": "淤泥泵",
+  "block.deepdrilling.sludge_pump.tooltip.behaviour1": "_提高采掘效率_的同时_减少钻头损耗_，但会降低采掘产物的_质量和数量_且积累_淤泥_。",
+  "block.deepdrilling.sludge_pump.tooltip.behaviour2": "连接的钻芯_停止工作_。",
+  "block.deepdrilling.sludge_pump.tooltip.condition1": "连接至钻芯时",
+  "block.deepdrilling.sludge_pump.tooltip.condition2": "淤泥蓄满时",
+  "block.deepdrilling.sludge_pump.tooltip.summary": "一种_钻探模块_，可与_钻芯_连接，多个连接时仅_其中一个_会生效",
+  "block.deepdrilling.veridium_node": "辉绿岩矿脉",
+  "block.deepdrilling.veridium_node.tooltip.behaviour1": "_辉绿岩，凝灰岩，安山岩_",
+  "block.deepdrilling.veridium_node.tooltip.condition1": "表面沉积物",
+  "block.deepdrilling.veridium_node.tooltip.summary": "在_海洋生物群系_的_基岩层_中生成",
+  "create.deepdrilling.recipe.ore_node": "矿脉钻探",
+  "deepdrilling.goggle.sludge_pump.backed_up": "备份",
+  "deepdrilling.loot.common": "零碎矿渣",
+  "deepdrilling.loot.earth": "伴生石料",
+  "deepdrilling.loot.rare": "富集矿石",
+  "deepdrilling.ponder.drill_basics.header": "进行深度钻探",
+  "deepdrilling.ponder.drill_basics.text_1": "给钻芯配备钻头并接入转速，钻芯会开始从矿脉中提取产物",
+  "deepdrilling.ponder.drill_basics.text_2": "使用材质更好的钻头可以加速采掘过程……",
+  "deepdrilling.ponder.drill_basics.text_3": "……当然，增加接入的转速也同样有效",
+  "deepdrilling.ponder.drill_modules.header": "使用模块",
+  "deepdrilling.ponder.drill_modules.text_1": "根据接入模块种类和数量的不同，钻芯的行为和属性也会产生变化",
+  "deepdrilling.ponder.drill_modules.text_2": "一枚钻芯最多可以连接5个模块",
+  "deepdrilling.ponder.drill_modules.text_3": "间断的模块无法生效",
+  "deepdrilling.ponder.tag.drilling": "深度钻探",
+  "deepdrilling.ponder.tag.drilling.description": "与从地脉深处提取产物物相关的模块……",
```

```suggestion
  "deepdrilling.ponder.tag.drilling.description": "与从地脉深处提取产物相关的模块……",
```
这个deep drilling也是模块吗
原文Components related to extracting resources from deep within the earth

---

我看不到您的回复，请检查您的回复是否在pending状态。
您可以参考https://github.com/orgs/community/discussions/10369 来放出pending状态的回复。

---

现在能看到了吗
前几次回复没确认也能看见 我以为不用确认

---

可以了

---

```suggestion
  "deepdrilling.ponder.tag.drilling.description": "与从地脉深处提取产物相关的各类组件……",
```
按组件翻吧

---

改了